api_spec.yaml

```diff
@@ -1636,52 +1636,6 @@ paths:
         '404':
           $ref: '#/components/responses/404'
         
-        '500':
-          $ref: '#/components/responses/500'  
-  
-  /recipe/handshake:
```

Comment:
actually, keep this API, but mark it as deprecated. What i originally meant was that this API should not be called from the latest version of the backend SDK